relay-profiling/src/android.rs

```diff
@@ -78,6 +90,17 @@ impl AndroidProfile {
         Ok(())
     }
 
+    fn set_transaction(&mut self, transaction: &TransactionMetadata) {
+        self.transaction_name = transaction.name.clone();
```

Comment:
I guess that's the only I thought I could copy the value. `Copy` trait can't be implemented for the `TransactionMetadata` type and since I'm passing a shared reference, that's kind of the only way the compiler would let me off the hook. Do you have another suggestion?

Reply:
Nope, sounds good to me. I was just curious.